controllers/depresolver/depresolver_test.go

```diff
@@ -1283,6 +1284,39 @@ func TestResolveLoggerEmptyValues(t *testing.T) {
 	assert.Equal(t, SimpleFormat, config.Log.Format)
 }
 
+func TestResolveConfigSplitBrainCheckEnabled(t *testing.T) {
+	// arrange
+	defer cleanup()
+	expected := predefinedConfig
+	expected.SplitBrainCheck = true
+	// act,assert
+	arrangeVariablesAndAssert(t, expected, assert.NoError)
+}
+
+func TestResolveConfigSplitBrainCheckNotSet(t *testing.T) {
+	// arrange
+	defer cleanup()
+	expected := predefinedConfig
+	expected.SplitBrainCheck = false
+	// act
+	// assert
+	// SplitBrainCheck is not set and expecting to resolve SplitBrainCheck = true and NoError
+	arrangeVariablesAndAssert(t, expected, assert.NoError, SplitBrainCheckKey)
+}
+
+func TestResolveConfigSplitBrainDisabledInvalid(t *testing.T) {
```

Comment:
Small suggestion here
```suggestion
func TestResolveConfigSplitBrainCheckDisabledInvalid(t *testing.T) {
```